TestFoundation/TestNSCompoundPredicate.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  TestNSCompoundPredicate.swift
+//  Foundation
+//
+//  Created by Kevin Lundberg on 12/11/15.
+//  Copyright © 2015 Apple. All rights reserved.
+//
```

Comment:
Ah yeah, missed that, good call. I'll handle that in the morning. 

--
Kevin Lundberg

> On Dec 11, 2015, at 1:44 AM, Thi <notifications@github.com> wrote:
> 
> In TestFoundation/TestNSCompoundPredicate.swift:
> 
> > @@ -0,0 +1,87 @@
> > +//
> > +//  TestNSCompoundPredicate.swift
> > +//  Foundation
> > +//
> > +//  Created by Kevin Lundberg on 12/11/15.
> > +//  Copyright © 2015 Apple. All rights reserved.
> > +//
> You should look at another Swift file in this project and add the same legal notice.
> 
> —
> Reply to this email directly or view it on GitHub.
>